website/docs/docs/contributing/documenting-a-new-adapter.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Documenting a new adapter"
+id: "documenting-a-new-adapter"
+---
+
+If you've already [built an adapter](/website/docs/docs/contributing/building-a-new-adapter.md), and [tested your adapter]((/website/docs/docs/contributing/testing-a-new-adapter.md)), it's time to document it so the dbt community will know that it exists and how to use it!
+
+## General Guidelines
+
+### Single Source of Truth
+
+We have asked our partner adapter maintainers to use the [docs.getdbt.com repo](https://github.com/dbt-labs/docs.getdbt.com) (i.e. this site) as the single-source-of-truth for documentation rather than having to maintain the same set of information in three different places. Instead, we ask that the adapter repo's `README.md` and the dataplatform's documentation pages simply link to the corresponding page on this docs site.
+
+### Assumed Knowledge
+
+To simplify things, assume the reader of this documentation already knows how both dbt and your data platform works. There's already great material for how to learn dbt and ostensibly for the adapter's back end data platform.
+
+
+## Topics and Pages to Cover
+
+
+
+| How To...            | File to change within `/website/docs/`                       | Action | Info to Include                                                                                                                                                                                      |
+|----------------------|--------------------------------------------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Connect              | `reference/warehouse-profiles/{MY-DATA-PLATOFRM}-profile.md` | Create | Give all information needed to define a target in `~/.dbt/profiles.yml` and get `dbt debug` to connect to the database successfully. All possible configurations should be mentioned.                |
+| Configure            | `reference/resource-configs/{MY-DATA-PLATOFRM}-configs.md`   | Create | What options and configuration specific to your data platform do users need to know? e.g. table distribution and indexing options, column_quoting policy, which incremental strategies are supported |
+| Discover and Install | `docs/available-adapters.md`                                 | Modify | Is adapter vendor- or community- supported? How to install Python adapter package? Ideally with pip and PyPI hosted package, but can also use `git+` link to GitHub Repo                             |
+
+The following three pages need to be modified or created to have your data platform be listed on our documentation. After the corresponding pull request is merged, we ask that you link to these pages from your adapter repo's `REAMDE` as well as from your product documentation
```

Comment:
Could we move this to be above the table? Reading this quickly, it's not clear to me that I need to create or modified these new packages specifically in the folders in the repo. So kinda more words to be lear that you are going to be creating say a new profile page for just your data platform in the `warehouse-profiles` folder.

Reply:
done! lmk what you think now